src/_help/hubs/index.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Hubs
+---
+
+- TOC
+{:toc}
+
+## What is a hub
+
+Hubs are a feature available starting from our paid plans: they allow you to manage collections of API documentation simply and efficiently.
+
+With hubs, your API ecosystem is displayed on a single page, where documentation can be sorted by categories.
+
+Hubs also provide a unified changelog, making it easy to identify all changes across all the documentation it contains at a glance. Search is also global, allowing you to search across all documentation.
+
+## Create a hub
+
+In your dashboard, click on Hubs then on Create a hub.
+
+![](/images/help/create-hub.png)
+
+You will be asked to choose a name for the hub, which will be visible to everyone. The slug is the portion of the URL that you can customize. You can also choose to use a custom domain instead. Finally, you can decide whether this hub is visible to everyone or only to members of your organization.
+
+![](/images/help/hub-creation.png)
+
+## Add documentation to a hub
+
+### Using the webapp
+#### Deploy a new documentation
+From the "Hubs" tab, click on "Create Documentation" to add it to the desired hub. You can then specify the documentation's name, its slug, and its access level (Public or private).
+#### Transfer an existing documentation
+From the "Standalone Documentation" tab, select the relevant documentation and go to its settings. The "Hubs Settings" section allows you to choose which hub to associate this documentation with. You can also specify a category if necessary.
+
+![](/images/help/move-standalone-doc.png)
+### Using the CLI
+
+The CLI allows you to deploy your new documentation versions easily, either by using the token of the target documentation or the token of the hub. You can find these tokens in their respective settings.
+
+[Find here the commands and more information about the CLI.](/help/bump-cli)
+### Using the API
```

Comment:
I would add the same disclaimers than before on what's possible/not possible to do via the API.

I would also give a quick explanation that they'll be using the "hub" parameter of the POST request, and point more precisely to that parameter in the doc (with our cool highlight effect) https://developers.bump.sh/operation/operation-post-versions#operation-post-versions-body-hub

Reply:
:+1: